submissions/kosh3n/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hooli-style Popup</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link type="image/png" sizes="16x16" rel="icon" href="https://img.icons8.com/fluency/48/000000/circled-menu.png" />
+</head>
+
+<body>
+    <div class="wrapper">
+        <header class="header">
+            <div class="logo-conatiner">
+                <a href="#home"><img class="homo-img" src="https://img.icons8.com/parakeet/96/000000/experimental-home-parakeet.png" alt="home"></a>
+            </div>
+            <div class="welcome">
+                <p>Welcome to our site!</p>
+            </div>
+            <input class="popup_checkbox" type="checkbox" name="popup_checkbox" id="popup_checkbox" tabindex="1">
+            <label for="popup_checkbox" class="popup_checkbox_label">
+                <img class="popup_button" src="https://img.icons8.com/stickers/75/000000/menu.png" alt="">
+            </label>
+            <nav class="popup">
+                <ul class="nav-menu">
+                    <li class="nav-item" tabindex="2"><a href="#mail" class="nav-link">
+                            <figure class="figure">
+                                <img class="menu-img" src="https://img.icons8.com/stickers/100/000000/mail-contact.png" alt="mail">
+                                <figcaption class="figcaption">Mail</figcaption>
+                            </figure>
+                        </a></li>
+                    <li class="nav-item " tabindex="2"><a href="#" class="nav-link">
+                            <figure class="figure">
+                                <img class="menu-img" src="https://img.icons8.com/stickers/100/000000/video.png" alt="video">
```

Comment:
For icons it's better set an empty alt attribute, read more [here](https://accessibility.psu.edu/images/imageshtml/#empty)
Please, check All icons in your project